rllib/algorithms/tests/test_algorithm_config.py

```diff
@@ -72,18 +72,19 @@ def test_rollout_fragment_length(self):
             AlgorithmConfig()
             .rollouts(
                 num_rollout_workers=4,
-                num_envs_per_worker=3,
+                num_envs_per_worker=2,
                 rollout_fragment_length="auto",
             )
-            .training(train_batch_size=2456)
+            .training(train_batch_size=2500)
```

Comment:
I had to change this test, because this was testing something that is wrong imo.